static/js/settings_org.js

```diff
@@ -778,6 +682,8 @@ exports.build_page = function () {
 
     function get_complete_data_for_subsection(subsection) {
         let data = {};
+        const config = settings_config;
```

Comment:
Does this need a second name?

Reply:
It's a workaround for the linter complaining about long lines.

Reply:
I see. Maybe use more wrapping?

```js
                data.message_content_edit_limit_seconds =
                    settings_config.msg_edit_limit_dropdown_values[
                        edit_limit_setting_value
                    ].seconds;
```

Reply:
Part of the problem here is that `get_complete_data_for_subsection` is nested inside of `build_page`, which I think is completely unnecessary.  I kinda want to avoid the rabbit hole of cleaning up everything that's ugly about the settings system code.

Reply:
I added a comment for this.  I agree with Steve that the right path forward here involves de-nesting the ~7 functions that are weirdly nested inside `build_page`.